src/helpers/wallet.helper.js

```diff
@@ -83,6 +86,7 @@ export function getWalletConfigFromSeed({
     password: '123',
     pinCode: '123',
```

Comment:
[🚫 no entry] It is with the default value. Should we remove in this PR or another PR will solve it?